WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationStep.swift

```diff
@@ -0,0 +1,18 @@
+final class SiteInformationStep: WizardStep {
+    private let creator: SiteCreator
+
+    private(set) lazy var content: UIViewController = {
+        return SiteInformationWizardContent(completion: didGoNext)
+    }()
+
+    var delegate: WizardDelegate?
+
+    init(creator: SiteCreator) {
+        self.creator = creator
+    }
+
+    private func didGoNext(_ data: SiteInformation) {
+        creator.information = data
+        delegate?.wizard(self, willNavigateTo: WebAddressStep.identifier)
```

Comment:
I'm noticing that none of the steps call `WizardDelegate.wizard(_:didNavigateTo:)` so far. 

Do you think that's a candidate for removing from the protocol, or should be be called in the initializer above?

Reply:
Also, should the individual steps be telling the wizard where to go next? It seems like the steps should be advising the wizard that the completed successfully (or not), deferring to the wizard where to go next.

Reply:
Yes, I agree. It did make sense to pass the identifier when there seemed like the navigation might not be completely sequential, but now, it doesn't anymore